src/app/RecommendationsList/lookForAddElChboxUse.js

```diff
@@ -0,0 +1,34 @@
+import { GenerateRecomendationsSourceList } from "./recomentadionsSource"
+import { RecomendationsSourceRenderer } from "./recomendationSourceRenderer";
+export class AddElementCheckbox {
+    constructor() {
+        this.checkboxBtn = checkboxCheck
+    }
+    static lookForCheckboxUse(requestResult) {
```

Comment:
use more meaningful method names, it needs to tell a story of what's happening, really doesnt need to be concise, if its 144 characters long but descriptive, go for it, same goes for variable names